go/src/koding/config/config.json

```diff
@@ -55,6 +55,50 @@
         "url": "https://dev-p2.koding.com/-/ipcheck"
       }
     ],
+    "kd-latest": [
```

Comment:
Actually this part is done, I will move all these variables to main.{}.coffee. Haven't done this yet because I'm not sure if there is main.prod.coffee in credential(?) repo which will also require updates in that case.